tensorflow_lite_support/python/test/task/vision/image_classifier_test.py

```diff
@@ -128,6 +164,129 @@ def test_classify_model(self, model_file_type, max_results,
     self.assertDeepAlmostEqual(
         image_result_dict, expected_result_dict, places=5)
 
+  def test_classify_model_with_bounding_box(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, max_results=3)
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Bounding box in "burger.jpg" corresponding to "burger_crop.jpg".
+    bounding_box = bounding_box_pb2.BoundingBox(
+      origin_x=0, origin_y=0, width=400, height=325)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
+      {'index': 934, 'score': 0.8815076351165771, 'class_name': "cheeseburger"},
+      {'index': 925, 'score': 0.019456762820482254, 'class_name': "guacamole"},
+      {'index': 932, 'score': 0.012489477172493935, 'class_name': "bagel"}
+    ]
+
+    # Builds test data.
+    expected_result_dict = self.build_test_data(expected_categories)
+
+    # Comparing results (classification w/ bounding box).
+    self.assertDeepAlmostEqual(image_result_dict, expected_result_dict, places=5)
+
+  def test_score_threshold_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, score_threshold=0.5)
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
+      {'index': 934, 'score': 0.7399742007255554, 'class_name': "cheeseburger"}
+    ]
+
+    # Builds test data.
+    expected_result_dict = self.build_test_data(expected_categories)
+
+    # Comparing results.
+    self.assertDeepAlmostEqual(image_result_dict, expected_result_dict, places=5)
+
+  def test_allowlist_option(self):
+    # Creates classifier.
+    model_file = _ExternalFile(file_name=self.model_path)
+
+    classifier = self.create_classifier_from_options(
+      model_file, class_name_allowlist=['cheeseburger', 'guacamole'])
+
+    # Loads image.
+    image = tensor_image.TensorImage.from_file(self.test_image_path)
+
+    # Classifies the input.
+    image_result = classifier.classify(image, bounding_box=None)
+    image_result_dict = json.loads(json_format.MessageToJson(image_result))
+
+    # Expected results.
+    expected_categories = [
```

Comment:
As you're testing the allowlist, you should assert if there's all class names in image_result belongs to the class_name_allowlist, rather than comparing it with a predefined result.